content/en/community/contributing/core-maintainer.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Core Maintainer"
+linkTitle: "Core Maintainer"
+weight: 10
+description: >
+  Who can commit directly to CHT repositories in GitHub
+---
+
+Historically only [Medic](https://medic.org/) teammates were given GitHub permissions to merge code into CHT repositories.  This is no longer the case!  Now, everyone is treated as an equal:  If you meet these requirements, you can commit code.  Medic teammates who already have maintainer status do not get exemptions - they've just already meet the requirements.
+
+This is not just for the [CHT Core](https://github.com/medic/cht-core/) repository though!  We welcome maintainers want to [update the documentation](https://docs.communityhealthtoolkit.org/) as well.  There's actually dozens of repositories that you can contribute to - [check them out](https://github.com/orgs/medic/repositories)!
+
+If you have any questions about how this works in practice, please post a [question to the forum](https://forum.communityhealthtoolkit.org/c/community/10).
+
+## Requirements
+
+* Participate in a squad where you author, commit and revise code that gets merged to at least one CHT repo *<br/>
+  **OR** <br/>
+  3 features merged to CHT Core outside of a squad *<br/>
+  **OR** <br/>
+  5 bug fixes merged to CHT Core outside of a squad *
+* Have a forum account and have achieved the "[Member](https://forum.communityhealthtoolkit.org/badges/2/member)" badge. 
+* Attend 3 out of 12 round up calls a year 
+* Have 2 existing core maintainers sign off on your nomination 
```

Comment:
I believe a core maintainer should also have experience in reviewing code, not just contributing code.